client/src/components/DayView/Calendar.js

```diff
@@ -0,0 +1,92 @@
+// import interactionPlugin from '@fullcalendar/interaction'; // needed for dayClick
+// import dayGridPlugin from '@fullcalendar/daygrid';
+import FullCalendar from '@fullcalendar/react';
+import resourceTimeGridPlugin from '@fullcalendar/resource-timegrid';
+import { message } from 'antd';
+import React from 'react';
+import './style.css';
+
+class Calendar extends React.Component {
+  calendarComponentRef = React.createRef();
+
+  state = {
+    rooms: [],
+    events: [],
+  };
+
+  componentDidMount() {
+    this.fetchRoomName().then(() => this.fetchRoomEvent('2020-04-14'));
+  }
+
+  fetchRoomName = () =>
+    fetch(`/api/v1/rooms`)
+      .then((res) => {
+        if (!res.ok) {
+          res.json().then(({ message: msg }) => message.error(msg));
+          throw res.statusText;
+        }
+        return res.json();
+      })
+      .then((results) => {
+        this.setState({ rooms: results });
+      })
+      .catch(() => {
+        message.error('error');
+      });
+
+  fetchRoomEvent = (date) => {
+    fetch(`/api/v1/rooms/${date}`)
+      .then((res) => {
+        if (!res.ok) {
+          res.json().then(({ message: msg }) => message.error(msg));
```

Comment:
this can be 404, bc there are no events on that day, either change the backend status code, or handle it here